omnigibson/systems/micro_particle_system.py

```diff
@@ -1680,6 +1613,7 @@ def clothify_mesh_prim(cls, mesh_prim, remesh=True, particle_distance=None):
                     # Terminate anyways, but don't fail
                     log.warn("Failed to sufficiently remesh cloth. "
                              "The generated cloth may not have evenly distributed particles.")
+                    break
```

Comment:
Regarding `avg_edge_percentage_mismatch`, I wonder whether checking "avg_edge_length" is enough. Ultimately we want the edges to have similar length and that length is close to `particle_distance`. Maybe we should use that stop criterion instead?

Reply:
added todo in asana, leave for future PR